### PR TITLE
Logging and Timing

### PR DESCRIPTION
**Pull request by jorenham**
### Logging

https://github.com/ProgrammingLife5/LifeTiles/issues/102 Created the _Logging_ utility class. Example usage:

``` java
try {
    methodThatThrows();
} catch (SomeException e) {
    Logging.exception(e);
}
```

---
### Timer

 https://github.com/ProgrammingLife5/LifeTiles/issues/120 Created the _Timer_ utility class that is able to directly log the results Example usage:

``` java
Timer timer = Timer.getAndStart();
peelThePotato();
timer.stopAndLog("Potato peeling");
```

This will log `Potato peeling took 666 ms`

---

**This pull request will be merged 23.59 31-05-2015**

### CODE REVIEW

**Comment by joswinter** [comment]
Looks very good, this shuts up pmd nicely!

File: src/main/java/nl/tudelft/lifetiles/Launcher.java [hunk 1]
```diff
@@ -50,7 +51,7 @@
      * @param args
      *            The program arguments
      */
-    public static void main(final String[] args) {
+    public static void main(final String... args) {
```
Comment (jorenham) [inline src/main/java/nl/tudelft/lifetiles/Launcher.java:54]: I'm sorry, this technically shouldn't have been in this pull-request, but I got annoyed by PMD.

Comment (joswinter) [inline src/main/java/nl/tudelft/lifetiles/Launcher.java:54]: Yeah, I didn't saw that one before so I looked it up. It seems to be a working alternative.

**Comment by alangerak** [comment]
According to http://stackoverflow.com/questions/1770010/how-do-i-measure-time-elapsed-in-java, when you only want to measure time it better to use System.currentTime vs Calender because the latter is expensive to use. 
Also, is it possible to have more than 1 logfile? Because if you run the program multiple times to check for a certain error you lose potential value information if there is only 1 file

**Comment by jorenham** [comment]
@alangerak It is possible to have multiple logfiles, but you'll have to to create another logger manually (which isn't hard) because the Logging class is static.

**Comment by albert-smit** [comment]
looks good

**Comment by alangerak** [comment]
Looks good now

**Pull request merged**